src/languages/scala.js

```diff
@@ -137,6 +137,25 @@ export default function(hljs) {
     }
   };
 
+  const IMPORT_EXPORT = {
+    beginKeywords: 'import export',
+    end: /[;\n]/,
+    excludeEnd: true,
```

Comment:
Ditto for this.

Reply:
You resolved this, but it's unchanged.  Ditto on the above as well. Unresolving.

Reply:
Removed this too.